debezium-connector-postgres/src/test/java/io/debezium/connector/postgresql/AbstractRecordsProducerTest.java

```diff
@@ -546,6 +546,8 @@ protected void assertRecordOffset(SourceRecord record, boolean expectSnapshot, b
         assertNotNull(offset.get(SourceInfo.TXID_KEY));
         assertNotNull(offset.get(SourceInfo.TIMESTAMP_KEY));
         assertNotNull(offset.get(SourceInfo.LSN_KEY));
+        assertNotNull(offset.get(SourceInfo.SCHEMA_NAME_KEY));
```

Comment:
How about actually asserting the DB, schema and table names here? It should be possible to pass a `TableId` with the expected attributes to this method.